assets/javascripts/modules/analytics/setup.js

```diff
@@ -4,25 +4,24 @@ define([
     'modules/analytics/omniture',
     'modules/analytics/remarketing',
     'modules/analytics/krux',
-    'modules/analytics/affectv'
-], function (
-    cookie,
-    ga,
-    omniture,
-    remarketing,
-    krux,
-    affectv
-) {
+    'modules/analytics/affectv',
+    'modules/analytics/snowplow'
+], function (cookie,
+             ga,
+             omniture,
+             remarketing,
+             krux,
+             affectv,
+             snowplow) {
     'use strict';
 
     function init() {
         var analyticsEnabled = (
-            window.guardian.analyticsEnabled &&
-            !navigator.doNotTrack &&
-            !cookie.getCookie('ANALYTICS_OFF_KEY')
+            window.guardian.analyticsEnabled && !navigator.doNotTrack && !cookie.getCookie('ANALYTICS_OFF_KEY')
```

Comment:
This is my annoying intellij auto reformat settings, I agree this was more readable before